src/mds/CDir.cc

```diff
@@ -2562,6 +2559,11 @@ void CDir::_omap_commit(int op_prio)
     string key;
     dn->key().encode(key);
 
+    dout(1) << *dn << " " << dn->corrupt_first_loaded << dendl;
+    if (!dn->corrupt_first_loaded) {
+      dn->check_corruption(false);
+    }
```

Comment:
Is this for debug only ? The debug level is `1` ?

Reply:
... yes!

Good catch!